buildpacks/ruby/src/layers/metrics_agent_install.rs

```diff
@@ -153,6 +158,15 @@ impl Layer for MetricsAgentInstall {
     }
 }
 
+// Check integrity of download, `sha_some` is awesome.
+fn sha_some(path: &Path) -> Result<String, std::io::Error> {
```

Comment:
Can we just use https://github.com/heroku/libcnb.rs/blob/5e9aeb916d1706d81b90948913050f6589726c87/libherokubuildpack/src/digest.rs?

Reply:
Are you asking me to make my code **less** `sha_some` :p 

Funny that it has the exact same signature. I didn't have `libherokubuildpack` pulled into this project, but I'll move to use the shared tooling 🎉 thanks for the recommendation!